biolink-model.yaml

```diff
@@ -5017,17 +5184,36 @@ slots:
     inverse: catalyzes
 
   has substrate:
-    is_a: has participant
+    description: >-
+      Carried by a transporter, possibly competing with the natural substrate for transport.
+    is_a: molecular modification decreased by
     # domain: molecular activity
     # range: chemical entity
     annotations:
       tag: biolink:canonical_predicate
       value: true
+    exact_mappings:
+      - NCIT:C120264
+    close_mappings:
+      - CHEMBL.MECHANISM:substrate
+      - DRUGBANK:substrate
+      - Inxight:substrate∂
+      - CTD:substrate
+      - GNBR:substrate
 
   is substrate of:
-    is_a: participates in
+    is_a: decreases molecular modification of
     inverse: has substrate
 
+  has product:
+    description: >-
+      The end result of a manufacturing process; anything that is produced.
```

Comment:
Definition here seems at odds with application to molecular products of catalyzed reactions